tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -172,3 +176,259 @@ def match_histograms(image_a, image_b, kernel_size=7):
         image_a = exposure.match_histograms(image_a, image_b).astype(np.uint8)
 
     return image_a, image_b
+
+
+class DFBRegistration:
+    r"""Deep Feature based Registration
+
+    This class implements a CNN feature based registration,
+    as proposed in a paper titled `Deep Feature based Cross-slide Registration
+    <https://arxiv.org/pdf/2202.09971.pdf>`_.
+
```

Comment:
IS it possible to format this reference as in other docstrings? @shaneahmed do you know a good example?

Reply:
Like this?
https://github.com/TissueImageAnalytics/tiatoolbox/blob/810a0ee7cdb70b6a554c4f910bab0197d5575074/tiatoolbox/models/architecture/idars.py#L53